pocs/tests/test_astrohaven_dome.py

```diff
@@ -0,0 +1,75 @@
+# Test the Astrohaven dome interface using a simulated dome controller.
+
+import copy
+import pytest
+import serial
+
+from pocs.dome import CreateDomeFromConfig
```

Comment:
This should be `create_dome_from_config` but I couldn't comment on file up above and must have skipped it in previous review.

Reply:
Done.